lib/mayaUsd/fileio/jobs/jobArgs.cpp

```diff
@@ -430,6 +430,111 @@ std::set<unsigned int> _FilteredTypeIds(const VtDictionary& userArgs)
     return result;
 }
 
+std::vector<VtValue>
+_MergeVectors(const std::vector<VtValue>& existingValues, const std::vector<VtValue>& newValues)
+{
```

Comment:
These new function are refactored code from the previously existing function named _MergeJobContexts, fiund below.